give.php

```diff
@@ -356,6 +356,8 @@ private function setup_constants() {
 		private function includes() {
 			global $give_options;
 
+			require_once ABSPATH . 'wp-admin/includes/plugin.php';
```

Comment:
@ravinderk This file is required for `is_plugin_active()` to work. It is not working without this file.